fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/DataApiRequestImpl.java

```diff
@@ -371,14 +372,14 @@ protected Boolean isDateTimeFirstSortField(LinkedHashMap<String, SortDirection>
     public DataApiRequestImpl() {
         super();
         this.table = null;
-        this.granularity = null;
-        this.dimensions = null;
+        this.granularity = DefaultTimeGrain.DAY;
```

Comment:
I'm not sure giving granularity a value that might appear correct is reasonable here.

It's one thing to make empty collections instead of nulls, it's a different thing to make a deliberate value.
Leave DefaultTimeGrain.DAY null, and if that breaks some things, let's see what the cost of fixing them